lms/djangoapps/student_account/views.py

```diff
@@ -247,6 +264,11 @@ def update_context_for_enterprise(request, context):
     else:
         context['enable_enterprise_sidebar'] = False
 
+    if sync_learner_profile_data:
+        context['data']['hide_auth_warnings'] = True
+        enterprise_customer = enterprise_customer_for_request(request)
+        context['data']['enterprise_name'] = enterprise_customer and enterprise_customer['name']
```

Comment:
I don't think you need this. We are already populating these values above in `enterprise_sidebar_context`.